lib/bundler/feature_flag.rb

```diff
@@ -44,6 +44,7 @@ def self.settings_method(name, key, &default)
     settings_flag(:init_gems_rb) { bundler_2_mode? }
     settings_flag(:list_command) { bundler_2_mode? }
     settings_flag(:lockfile_uses_separate_rubygems_sources) { bundler_2_mode? }
+    settings_flag(:major_deprecations) { bundler_2_mode? }
```

Comment:
IMO this should not be a feature flag as this is not enabling any particular Bundler feature and would also never be removed either. Can we just set the config option to have a default value instead?

Reply:
That makes sense to me, I'm going to change it as you suggest :+1:.